src/test/java/org/mskcc/cbio/portal/integrationTest/dao/TestDaoMutation.java

```diff
@@ -58,7 +57,9 @@
 import java.util.Map;
 import java.util.Set;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Which assertEquals are you using?
Do we need to have both?

Reply:
No, I don't need to have both. Good catch, thanks!